src/library.jl

```diff
@@ -777,7 +777,9 @@ complete(rf::R_{FlagFirst}, result) = complete(inner(rf), unwrap(rf, result)[2])
     Partition(size, step = size, flush = false)
     Partition(size; step = size, flush = false)
 
-Sliding window of width `size` and interval `step`.
+Sliding window of width `size` and interval `step`. Note: because 
+`step` = `size` is the default, hence the default behaviour is 
+non-overlapping windows.
```

Comment:
```suggestion
Sliding window of width `size` and interval `step`.

Note: `step` = `size` is the default.  Hence, the default behavior is
non-overlapping windows.
```

Reply:
The first paragraph would be in the [List of transducers table](https://juliafolds.github.io/Transducers.jl/dev/#List-of-transducers). So I prefer to keep it very short (although sometimes I forget to/cannot do it).

I also did a very minor tweak.